node/runtime/pangoro/src/pallets/bridge_messages.rs

```diff
@@ -1,56 +1,47 @@
 pub use pallet_bridge_messages::Instance1 as WithPangolinMessages;
 
-// --- paritytech ---
+// --- darwinia-network ---
+use crate::{bridges_message::bm_pangolin, *};
 use bp_messages::MessageNonce;
-use bp_pangolin::{
-	AccountIdConverter, MAX_SINGLE_MESSAGE_DELIVERY_CONFIRMATION_TX_WEIGHT,
-	MAX_UNCONFIRMED_MESSAGES_IN_CONFIRMATION_TX, MAX_UNREWARDED_RELAYERS_IN_CONFIRMATION_TX,
-};
 use bp_runtime::{ChainId, PANGOLIN_CHAIN_ID};
-use pallet_bridge_messages::Config;
-// --- darwinia-network ---
-use crate::*;
 use darwinia_fee_market::s2s::{
 	FeeMarketMessageAcceptedHandler, FeeMarketMessageConfirmedHandler, FeeMarketPayment,
 };
 use darwinia_support::evm::{ConcatConverter, IntoAccountId, IntoH160};
-use pangolin_messages::{
-	FromPangolinMessageDispatch, FromPangolinMessagePayload, Pangolin,
-	PangoroToPangolinMessagesParameter, ToPangolinMessagePayload, ToPangolinMessageVerifier,
-};
+use pallet_bridge_messages::Config;
 
 frame_support::parameter_types! {
 	pub const MaxMessagesToPruneAtOnce: MessageNonce = 8;
 	pub const MaxUnrewardedRelayerEntriesAtInboundLane: MessageNonce =
-		MAX_UNREWARDED_RELAYERS_IN_CONFIRMATION_TX;
+		bp_pangolin::MAX_UNREWARDED_RELAYERS_IN_CONFIRMATION_TX;
 	pub const MaxUnconfirmedMessagesAtInboundLane: MessageNonce =
-		MAX_UNCONFIRMED_MESSAGES_IN_CONFIRMATION_TX;
+		bp_pangolin::MAX_UNCONFIRMED_MESSAGES_IN_CONFIRMATION_TX;
 	// `IdentityFee` is used by Pangoro => we may use weight directly
 	pub const GetDeliveryConfirmationTransactionFee: Balance =
-		MAX_SINGLE_MESSAGE_DELIVERY_CONFIRMATION_TX_WEIGHT as _;
+		bp_pangoro::MAX_SINGLE_MESSAGE_DELIVERY_CONFIRMATION_TX_WEIGHT as _;
 	pub RootAccountForPayments: Option<AccountId> = Some(ConcatConverter::<_>::into_account_id((&b"root"[..]).into_h160()));
 	pub const BridgedChainId: ChainId = PANGOLIN_CHAIN_ID;
 }
 
 impl Config<WithPangolinMessages> for Runtime {
 	type Event = Event;
 	type WeightInfo = ();
-	type Parameter = PangoroToPangolinMessagesParameter;
+	type Parameter = bm_pangolin::PangoroToPangolinMessagesParameter;
 	type MaxMessagesToPruneAtOnce = MaxMessagesToPruneAtOnce;
 	type MaxUnrewardedRelayerEntriesAtInboundLane = MaxUnrewardedRelayerEntriesAtInboundLane;
 	type MaxUnconfirmedMessagesAtInboundLane = MaxUnconfirmedMessagesAtInboundLane;
 
-	type OutboundPayload = ToPangolinMessagePayload;
+	type OutboundPayload = bm_pangolin::ToPangolinMessagePayload;
 	type OutboundMessageFee = Balance;
 
-	type InboundPayload = FromPangolinMessagePayload;
-	type InboundMessageFee = pangolin_primitives::Balance;
-	type InboundRelayer = pangolin_primitives::AccountId;
+	type InboundPayload = bm_pangolin::FromPangolinMessagePayload;
+	type InboundMessageFee = bp_pangolin::Balance;
+	type InboundRelayer = bp_pangolin::AccountId;
 
-	type AccountIdConverter = AccountIdConverter;
```

Comment:
The original `type AccountIdConverter = bp_pangolin::AccountIdConverter;` seems wrong.

Reply:
We made a mistake before.